Yoast/wpseo-woocommerce
# Rename class files

## Summary
This PR can be summarized in the following changelog entry:
_N/A_

## Relevant technical choices:
* No functional changes.
* Code style compliance.

As discussed with moorscode.

Adjusts the file names to be in line with the rules as laid out in the `Yoast.Files.FileName` sniff.
Ref: Yoast/yoastcs#48

Simple renames:
* Remove `class-` prefixes from the plugin file names.
* Remove `wpseo-`/`yoast-` prefixes from the plugin file names.
* Remove `test-class-` prefixes from the test files and add `-test` suffix.
* Make sure that the file names accurately reflect the class names (excluding the prefixes).

The second commit adjusts where necessary references to file names througout the plugin (only two).

- [ ] **Note**: The classmap autoload files for Composer will need to be regenerate!

## Test instructions

This PR can be tested by following these steps:

* See if the plugin still loads as expected, unit tests still run as expected etc.
